### DISCUSSION TITLE
Cannot find module in monorepo

### DISCUSSION THREAD

**Discussion by chav-aniket** [swc-project/swc#5590] [Q&A]
Hello, I'm using turborepo for a node application and trying to set up swc to build my application. My app is dependent on a local package as part of the monorepo and I've initialised how to build both the dependency and the app, and they build without issues. When I try to run the app however I get the "Cannot find module" error for my local dependency. 
As according to the folder structure below, my microservice depends on my utils package. It's added to the microservice's package.json all fine and everything works in dev servers, just need to sort out how to build and run a production server. 
Is this a bundling issue? Are there any suggestions for how I could fix this? Thank you!

```txt
# folder structure
apps
└── microservice
    ├── dist
    └── src
packages
├── eslint-config-custom
├── utils
└── tsconfig
```

```json
{
  "$schema": "https://json.schemastore.org/swcrc",
  "jsc": {
    "parser": {
      "syntax": "typescript",
      "tsx": false,
      "dynamicImport": true,
      "decorators": true,
      "importAssertions": true
    },
    "transform": {
      "decoratorMetadata": true
    },
    "target": "es2015",
    "loose": true,
    "externalHelpers": true
  },
  "module": {
    "type": "commonjs",
    // These are defaults.
    "strict": false,
    "strictMode": true,
    "lazy": false,
    "noInterop": false,
    "importInterop": "node"
  },
  "minify": true
}
```

**Comment by kdy1** [comment]
swc has nothing to do with module resolution by default. Check if `jsc.paths` works, and file an issue with a minimal repro if there's a bug
